task/kraken-scenario/0.1/README.md

```diff
@@ -0,0 +1,65 @@
+# Kraken-scenario
+
+The kraken-scenario task execute a chaos test against a cluster and return a OK if test is positive.
+
+## Install the Kraken-scenario task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/startxfr/tekton-catalog/main/task/kraken-scenario/0.1/kraken-scenario.yaml
+```
+
+## Parameters
+
+| Key                 | Default            | Description                                                                                                     |
+| ------------------- | ------------------ | --------------------------------------------------------------------------------------------------------------- |
+| awsCredentialSecret | `kraken-aws-creds` | Name of the secret key holding the aws-credentials (mandatory but used only for aws infrastructure chaos test). |
+
+## Workspaces
+
+| Key               | Mount                 | Description                                                                                                       |
+| ----------------- | --------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| kraken-kubeconfig | `/root/.kube`         | The kubeconfig configuration file of the targeted cluster. Must be in a `config` key if from configMap or Secret. |
+| kraken-config     | `/root/kraken/config` | The kraken configuration file defining the chaos scenario to run                                                  |
+| kraken-common     | `/root/kraken/common` | The kraken common configuration files related to the chaos configuration                                          |
+
+## Platforms
+
+The Task can be run on `linux/amd64` platform.
+
+## Usage
+
+This task should be run with a pre-existing context composed of various `secret` and `configMap` resources. 
+You better check the [samples files](https://github.com/startxfr/tekton-catalog/tree/main/task/kraken-scenario/0.1/samples) 
+instead of using this example _as it_.
+
+When you have created your 
+`kraken-aws-creds` Secret ([example](https://github.com/startxfr/tekton-catalog/tree/main/task/kraken-scenario/0.1/samples/kraken-aws-creds.yaml)),
+`kraken-common-example` configMap ([example](https://github.com/startxfr/tekton-catalog/tree/main/task/kraken-scenario/0.1/samples/kraken-common-example.yaml)),
+`kraken-config-example` configMap ([example](https://github.com/startxfr/tekton-catalog/tree/main/task/kraken-scenario/0.1/samples/kraken-config-example.yaml)) and
+`kraken-kubeconfig` configMap ([example](https://github.com/startxfr/tekton-catalog/tree/main/task/kraken-scenario/0.1/samples/kraken-kubeconfig.yaml)) you can
```

Comment:
instead of pointing the samples from your fork, can you please point the same from tektoncd/catalog? (As of now the hyperlinks won't work but once the PR is merged these will work so that's fine)

Reply:
done in https://github.com/tektoncd/catalog/pull/1003/commits/c718624f3c52ffa990b701f0e7ca130881f016ce